test/test_plugin_helper.rb

```diff
@@ -1,9 +1,11 @@
 require 'test_helper'
 
 def sample_facts_file
-  File.read(
-    File.join(
-      ForemanAnsible::Engine.root, 'test', 'fixtures', 'sample_facts.json'))
+  File.read(File.join(
```

Comment:
Nitpick, 

```ruby
File.read(
    File.join(
      ForemanAnsible::Engine.root,
      'test',
      'fixtures',
      'sample_facts.json'
    )
  )

```

reads a bit better IMO - also it's what's suggested in the newest Rubocop versions (0.42)